readme.adoc

```diff
@@ -1163,6 +1163,26 @@ launch faster on consecutive runs.
 
 The caching goes to `~/.jbang/cache` by default, you can run `jbang cache clear` to remove all cache data from this folder.
 
+== Install apps [EXPERIMENTAL]
+
+Since version 0.56 jbang comes with ability to setup jbang to put scripts/apps into your `PATH`
+using `jbang app`. This is useful to easily make scripts available from anywhere on any Operating System;
+
+To get started run `jbang app setup` this will on Windows modify your system wide `PATH` to include a folder managed by jbang.
+On bash/zsh based shells it will setup `PATH` in your `bashrc` or `zshrc` file(s.)
+
+Once setup, you can use `jbang app install <scriptRef>` to install that script in path.
```

Comment:
I think we need a "the" or "your" before "path", right?

Reply:
made it "into the jbang managed path"